docs/TESTS.md

```diff
@@ -4,9 +4,17 @@ We recommend you install [pytest](http://pytest.org/latest/) and
 [pytest-cache](http://pythonhosted.org/pytest-cache/). `pytest` is a testing
 tool that will give you more flexibility over running your unit tests.
 
+If you want to install the version of pytest that obeys Python 2.X syntax, then use `pip`:
```

Comment:
This might be better worded as: "If you want to install `pytest` for Python 2, then use `pip`:"?
Similar wording can be used for Python 3.